apps/civicsignalblog/src/payload/access/applications/research.js

```diff
@@ -1,6 +1,7 @@
-import { RESEARCH } from "../../lib/data/common/applications";
 import canAccessApplication from "../canAccessApplication";
```

Comment:
Wgy don't we use the alias for all imports?

Reply:
Cool I'll rectifty that one 👍